### ISSUE TITLE
[3.x] Not clear cache after upgrade MODX 2.7.2 to MODX 3.0.0-alpha1

### ISSUE THREAD

**Issue by CrazyBoy49z** [modxcms/revolution#14952]
### Summary
![image](https://user-images.githubusercontent.com/3448603/72474098-c1200000-37f0-11ea-9e8c-a16222c70353.png)

After manually clear "core/cache", works

### Step to reproduce
Upgrade MODX 2.7.2 to MODX 3.0.0-alpha1 with http/2

**Comment by JoshuaLuckers** [comment]
Are you sure this wasn't your web browser cache?

**Comment by CrazyBoy49z** [comment]
> Are you sure this wasn't your web browser cache?

Yes, I reload page with browser cache (ctrl+ F5). But I cleared the MODX cache, and the admin panel worked. @JoshuaLuckers check it one more time.

**Comment by OlegShchavelev** [comment]
Installed MODX 3.0 alpha 2. At the first authorization in CMS a lot of artifacts. To fix, you need to clear the cache 

![image](https://user-images.githubusercontent.com/20732384/73573172-c97c6a00-4483-11ea-89ab-07f2098808fd.png)

**Comment by Jako** [comment]
The lexicon cache has to be cleared automatically during an update. That should be the issue for the w_configcheck and other not replaced lexicon strings.

We should also think about a cache busting configuration for the browser assets.

**Comment by CrazyBoy49z** [comment]
> We should also think about a cache busting configuration for the browser assets.

I have idea.
For clear browser assets cache add MODX version to files. Example: style.css?3.0.0-alpha3

**Comment by CrazyBoy49z** [comment]
@Jako but the best way add with PHP function [filemtime](https://www.php.net/manual/en/function.filemtime.php)

**Comment by Mark-H** [comment]
Just refreshing the page seems to be enough to bring back the lexicons on a clean install, no need for clearing any caches beyond that.

**Comment by CrazyBoy49z** [comment]
@Mark-H this problem with browser cache need fix  #14988

**Comment by Jako** [comment]
@Mark-H If it is only a browser caching (refreshing) issue, this could be solved with cache busting too.
